src/display/editor/editor.js

```diff
@@ -276,6 +281,17 @@ class AnnotationEditor {
     this.fixAndSetPosition();
   }
 
+  translateInPage(x, y) {
+    const [width, height] = this.pageDimensions;
+    [x, y] = this.screenToPageTranslation(x, y);
+
+    this.x += x / width;
+    this.y += y / height;
+
+    this.fixAndSetPosition();
+    this.div.scrollIntoView({ block: "nearest" });
+  }
+
```

Comment:
This looks *almost* like a duplicate of the method just above, how about introducing a common helper to reduce the amount of code here?
Basically [replace this code](https://github.com/mozilla/pdf.js/pull/16746/files#diff-8ba8badc29ff13ad2d4b61ad2b7df503933c69c028150d1ba24188deb7f11493R269-R294) with something like
```javascript
  #translate([width, height], x, y) {
    [x, y] = this.screenToPageTranslation(x, y);
    this.x += x / width;
    this.y += y / height;
    this.fixAndSetPosition();
  }

  /**
   * Translate the editor position within its parent.
   * @param {number} x - x-translation in screen coordinates.
   * @param {number} y - y-translation in screen coordinates.
   */
  translate(x, y) {
    this.#translate(this.parentDimensions, x, y);
  }

  translateInPage(x, y) {
    this.#translate(this.pageDimensions, x, y);
    this.div.scrollIntoView({ block: "nearest" });
  }
```